### ISSUE TITLE
Create organization resources

### ISSUE THREAD

**Issue by alsaihn** [HTBox/crisischeckin#454]
Add database tables to allow an organization's point of contact to add resources to an organization. Resources should be simple description-and-count objects that link to a cluster id, similar to a commitment.

Depends on #451

**Comment by mjmilan** [comment]
@tonysurma This has been done, has it not?

**Comment by tonysurma** [comment]
@mjmilan I will review all of these with respect to what was code recently to see what is closed, what needs to be pushed further and what incurred technical debt (hacky choices, APIs in the wrong place etc., due to the speed of getting it in)

**Comment by andrewhart098** [comment]
@tonysurma There is a DB Resource table and associated migrations that have already been merged into master.  The Resource functionality specifics have been broken into other issues.  I am working on #473 and this existing Resource table/model seems to suffice.

**Comment by tonysurma** [comment]
Sounds right, closing this
